accounts/abi/bind/precompilebind/precompile_contract_test_template.go

```diff
@@ -139,4 +142,23 @@ func TestRun(t *testing.T) {
 	}
 	{{- end}}
 }
+
+func Benchmark{{.Contract.Type}}(b *testing.B) {
+	{{- if .Contract.AllowList}}
+	// Benchmark tests with allowlist tests.
+	// This adds allowlist run tests to your custom tests
```

Comment:
is `allowlist run` correct? also referenced above

Reply:
it is correct but not very obvious, so removed `run`